Assets/HoloToolkit/Input/Scripts/ControllerInfo.cs

```diff
@@ -219,5 +217,119 @@ public void LoadInfo(Transform[] childTransforms, ControllerVisualizer visualize
                 }
             }
         }
+
+        public void AnimateGrasp(bool isGrasped)
+        {
+            if (grasp != null && graspPressed != null && graspUnpressed != null && isGrasped != wasGrasped)
+            {
+                if (isGrasped)
+                {
+                    grasp.transform.localPosition = graspPressed.localPosition;
```

Comment:
Ooh good idea! It was definitely feeling a bit repetitive, but I hadn't thought of that.